services/frontend/pkg/config/config.go

```diff
@@ -151,3 +153,12 @@ type Checksums struct {
 	SupportedTypes      []string `yaml:"supported_types" env:"FRONTEND_CHECKSUMS_SUPPORTED_TYPES" desc:"Define the checksum types that indicate to clients which hashes the server can use to verify upload integrity. You can provide multiple types separated by blank or comma. Supported types are 'sha1', 'md5' and 'adler32'."`
 	PreferredUploadType string   `yaml:"preferred_upload_type" env:"FRONTEND_CHECKSUMS_PREFERRED_UPLOAD_TYPE" desc:"The supported checksum type for uploads that indicates to clients supporting multiple hash algorithms which one is preferred by the server. Must be one out of the defined list of SUPPORTED_TYPES."`
 }
+
+type PasswordPolicies struct {
+	MinCharacters            int    `yaml:"min_characters,omitempty" env:"OCIS_PASSWORD_POLICIES_MIN_CHARACTERS" desc:"Define the minimum password length. If a min_characters not set minimum password length is equal 1"`
+	MinLowerCaseCharacters   int    `yaml:"min_lower_case_characters" env:"OCIS_PASSWORD_POLICIES_MIN_LOWER_CASE_CHARACTERS" desc:"Define the minimum number of uppercase letters. The default value is equal 0"`
+	MinUpperCaseCharacters   int    `yaml:"min_upper_case_characters" env:"OCIS_PASSWORD_POLICIES_MIN_UPPER_CASE_CHARACTERS" desc:"Define the minimum number of lowercase letters. The default value is equal 0"`
+	MinDigits                int    `yaml:"min_digits" env:"OCIS_PASSWORD_POLICIES_MIN_DIGITS" desc:"Define the minimum number of digits. The default value is equal 0"`
+	MinSpecialCharacters     int    `yaml:"min_special_characters" env:"OCIS_PASSWORD_POLICIES_MIN_SPECIAL_CHARACTERS" desc:"Define the minimum number of special characters. The default value is equal 0"`
+	AllowedSpecialCharacters string `yaml:"allowed_special_characters" env:"OCIS_PASSWORD_POLICIES_ALLOWED_SPECIAL_CHARACTERS" desc:"Define the custom list of allowed special character and overwrite the default list."`
```

Comment:
What is the default behavior of this?

Reply:
* What is the default _list_, overwrite the default which is empty?
* Can you elaborate what a "special character is"?
We can also add a text part to the envvar as we do with others like `See the documentation for more details.` and describe it in the text.

Reply:
Added to the readme https://github.com/owncloud/ocis/pull/7195/files#diff-d75aa0c448ddadae4b29e6c34aa5f7331fbacb37d08d956e9c2b23eedd46d3e7R59
@dragonchaser Please validate if the default behavior is clear after reading.

Reply:
lgtm